src/hooks/useParticipatingVP.ts

```diff
@@ -0,0 +1,30 @@
+import { useMemo } from 'react'
+
+import useAsyncMemo from 'decentraland-gatsby/dist/hooks/useAsyncMemo'
+
+import { Snapshot, SnapshotProposal } from '../api/Snapshot'
+import { SNAPSHOT_SPACE } from '../entities/Snapshot/constants'
+import { groupProposalsByMonth, median } from '../entities/Snapshot/utils'
+
+export default function useParticipatingVP(start: Date, end: Date) {
+  const [proposals, state] = useAsyncMemo(
+    async () => {
+      // TODO: un-hardcode snapshot space
+      return await Snapshot.get().getProposals('snapshot.dcl.eth', start, end, ['created', 'scores_total'])
```

Comment:
It'd be great to update getProposals to not receive the space as an argument and use it from the constants file.

Reply:
TODO in a new PR as discussed